docs/pages/versions/unversioned/sdk/mail-composer.md

```diff
@@ -7,7 +7,7 @@ import InstallSection from '~/components/plugins/InstallSection';
 import PlatformsSection from '~/components/plugins/PlatformsSection';
 import Video from '~/components/plugins/Video'
 
-**`expo-mail-composer`** allows you to compose and send mail quickly and easily using the OS UI. This module can't be used on iOS Simulators since you can't sign into a mail account on them.
+**`expo-mail-composer`** allows you to compose and send mail (email or e-mail) quickly and easily using the OS UI. This module can't be used on iOS Simulators since you can't sign into a mail account on them.
```

Comment:
```suggestion
**`expo-mail-composer`** allows you to compose and send emails quickly and easily using the OS UI. This module can't be used on iOS Simulators since you can't sign into a mail account on them.
```